src/openloco/input.h

```diff
@@ -54,47 +54,47 @@ namespace openloco::input
     };
 
     void init();
-    void init_mouse();
-    bool has_flag(input_flags value);
-    void set_flag(input_flags value);
-    void reset_flag(input_flags value);
+    void initMouse();
```

Comment:
Annoyingly this is causing a colision with my PR as I did the functions in the input folder but input.h isn't in that folder. I'll probably merge this one then adjust my PR.